go.mod

```diff
@@ -227,8 +227,22 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.5.0 // indirect
 )
 
+require go.opentelemetry.io/contrib/otelconf v0.15.0
+
 require (
 	github.com/envoyproxy/go-control-plane/envoy v1.32.4 // indirect
+	github.com/golang/snappy v1.0.0 // indirect
 	github.com/google/btree v1.1.3 // indirect
 	github.com/prometheus/sigv4 v0.1.0 // indirect
+	go.opentelemetry.io/otel/exporters/otlp/otlplog/otlploggrpc v0.11.0 // indirect
```

Comment:
@swiatekm i think these are coming from the otelconf new dependency

Reply:
They do. I double checked.